JSTask2.js

```diff
@@ -0,0 +1,12 @@
+// There are two arrays with individual values, write a JavaScript program to compute the sum of each individual index value from the given arrays Sample array :
+// array1 = [1,0,2,3,4];
+// array2 = [3,5,6,7,8,13];
+// Expected Output : [4, 5, 8, 10, 12, 13]
+
+let array1 = [1,0,2,3,4];
+let array2 = [3,5,6,7,8,13];
+
+let sum = array1.map(function(num,id){
```

Comment:
![image](https://user-images.githubusercontent.com/22799565/211295997-087dfc4d-e971-41ae-a783-f056d61883d8.png)
Does not work for all conditions